analysis/cox-ipw.R

```diff
@@ -341,10 +339,14 @@ if (sum(episode_info[episode_info$time_period != "days_pre", ]$N_events) < total
   
   results$input <- opt$df_input
   
+  results$hr <- exp(results$lnhr)
+  results$conf.low <- exp(results$lnhr - qnorm(0.975)*results$se.lnhr)
+  results$conf.high <- exp(results$lnhr + qnorm(0.975)*results$se.lnhr)
+  
   results <- results[order(results$model),
                      c("model", "exposure", "outcome", "term",
-                       "estimate", "robust.conf.low", "robust.conf.high", "robust.se", "se",
-                       "N_total", "N_exposed", "N_events", "person_time_total",  "person_time_median",
+                       "lnhr","se.lnhr", "hr","conf.low", "conf.high", 
```

Comment:
Potentially could change the '.' to '_' in se.lnhr, conf.low, conf.high so they get classed as one word

Reply:
https://github.com/opensafely-actions/cox-ipw/pull/22/commits/b65c9c67190fb6304333efdabbc3dfd5432f2e53